frontend/src/components/eMIB/Overview.jsx

```diff
@@ -1,8 +1,37 @@
 import React, { Component } from "react";
+import "../../css/lib/aurora.min.css";
+import LOCALIZE from "../../text_resources";
+import "../../css/App.css";
+
+const styles = {
+  h2: {
```

Comment:
I think we may want to define this in App.css, since we will be using it for most of our h2s, no?

Reply:
This will be fixed in my next PR. In the build of TipOnTest component, I fixed that.

Reply:
Sounds good